make server-setup only link to OS-server-setup (#1066)
azerothcore/wiki: docs/linux-server-setup.md

| Installation Guide                                                                                                                   |                                                           |
| :----------------------------------------------------------------------------------------------------------------------------------- | :-------------------------------------------------------- |
| This article is a part of the Installation Guide. You can read it alone or click the previous link to easily move between the steps. |
| [<< Step 2: Core Installation](linux-core-installation)                                                                                    | [Step 4: Database Installation >>](database-installation) |

**Table of contents**
- [Data files](#data-files) - Required/Optional (Not needed if you extract the files.)
- [Extractors](#extractors) - Required/Optional (Not needed if you download the files.)
- [Config Files: Worldserver and Authserver](#config-files-worldserver-and-authserver) - Required

> Now that you have the source compiled, you need to add some necessary files, by either downloading or extracting them.

Reviewer: "some files"

> Now that you have the source compiled, you need to add some necessary files, by either downloading or extracting them.

Reviewer: ```suggestion
Now that you have the source compiled, you need to add the necessary client data. You can either download pre-extracted files or use the compiled extractors to extract the files yourself. Once the data is ready, you must update the `DataDir` option in your `worldserver.conf` file to point to the directory containing the data.
```

Some files are optional but highly recommended:

> | Directory |                    |
> | :-------- | :----------------- |
> | dbc       | Mandatory          |
> | maps      | Mandatory          |
> | vmaps     | HIGHLY RECOMMENDED |
> | mmaps     | HIGHLY RECOMMENDED |
> | cameras   | Recommended        |

Reviewer: ```suggestion
| Directory | Importance          | Purpose                                                                 |
|-----------|---------------------|-------------------------------------------------------------------------|
| dbc       | Mandatory           | Provides static game data for spells, items, creatures, and mechanics. |
| maps      | Mandatory           | Defines the terrain and layout of the game world.                      |
| mmaps     | Highly Recommended  | Provides advanced pathfinding for NPCs and creatures.                  |
| vmaps     | Highly Recommended  | Enables collision detection for buildings and objects.                 |
| cameras   | Recommended         | Enhances visual effects and camera angles for events.                  |

```

> ## Data files

Reviewer: Option 1: Download Pre-Extracted Files

If you intend to use an enUS client you can download the data files below. If you intend to use any other language client you will need to [extract](#extractors) the data yourself.

<a class="no-icon" href="https://github.com/wowgaming/client-data/releases/" target="_blank"><i class="fa-solid fa-download"></i> Data files enUS (AC Data v16)</a>

1. Download the files above.

> 2. Create a new folder within the build folder called **Data**. i.e **$HOME/azerothcore/data/**

Reviewer: ```suggestion
2. Create a new folder within the build folder called **data**. i.e **$HOME/azerothcore/data/**
```

3. Extract the files from the zip file and place them within the **Data** folder.

4. Edit your the [DataDir](#updating-datadir) config option to the location of your folder.

> ## Extractors

Reviewer: Option 2: Extract Files Yourself

Variablename = "MySQLIP;Port;Username;Password;database"  
``` 

The following steps must be verified:

- The hostname (127.0.0.1) can stay the same if AzerothCore is being installed on the same computer that you run WoW on.
  If not, follow the instruction in [Realmlist Table](realmlist).

- The port (3306) is the standard configured value. If you changed the default port in your MySQL settings, you must change it accordingly.
  The username and password can be variable. You can choose to either: 

    - use default acore / acore username and password pair.

    - create a unique login within a User Manager within your preferred database management tool (commonly identified by an icon that looks like a person or people) and give it the necessary permissions (SELECT, INSERT, UPDATE, DELETE permissions are sufficient, and is much safer).

### Updating DataDir

> 1. In your worldserver.conf file locate the **DataDir** option.

Reviewer: ```suggestion
1. In your `worldserver.conf` file locate the **DataDir** option.
```

> 1. Edit it to the path of your folder. i.e **$HOME/azerothcore/data/**

Reviewer: worldserver.conf has a generic example in the config. continue using our example folder here

```suggestion
1. Edit it to the path of your folder. e.g. **$AC_CODE_DIR/build/data**
```
